test/client/test.go

```diff
@@ -38,3 +40,24 @@ func (t *Test) Close() {
 		fmt.Printf("============\n\n")
 	}
 }
+
+//HackTest wraps the singleton test client for use with hack testing
```

Comment:
Call this "NamespaceClient" rather than "HackTest"? Its a client with a namespace that isn't related to any particular test.